code/game/objects/items/weapons/storage/pouches.dm

```diff
@@ -6,7 +6,7 @@
 	item_state = "pouch"
 
 	w_class = ITEM_SIZE_SMALL
-
+	slot_flags = SLOT_BELT //Pouches can be worn on belt
 	storage_slots = 1
 	max_w_class = ITEM_SIZE_SMALL
```

Comment:
How does this item make sense? It takes up `ITEM_SIZE_SMALL` slot and provides exactly 1 `ITEM_SIZE_SMALL` slot. In essence doing nothing.

Reply:
that item doesnt exist. It's the abstract base class and isn't obtainable ingame